src/models/serw13/README.md

```diff
@@ -58,8 +58,8 @@ The System76 Serval WS is a laptop with the following specifications:
         - Supports USB-PD (charging) when the system is powered off or suspended.
         - Supports DisplayPort over USB-C.
     - 1x USB 3.2 Gen 2 Type-C
-        - Supports DisplayPort over USB-C.
+<!---        - Supports DisplayPort over USB-C.--->
```

Comment:
I don't think this is the desired effect:
![2023-05-24_08-18-32](https://github.com/system76/tech-docs/assets/13512097/693ff0b6-36da-4c88-8cb7-1be4b6bb2a19)

Probably better to just delete the text, rather than trying to comment it out. Commenting things out of markdown isn't usually a recipe for a good time.

Reply:
Yes, that is not the desired change. Does it make sense to reject and close this PR and just start fresh? I don't think the change history *needs* a dead end change in there.

Reply:
I probably would have just fixed this one, amended the commit, and force pushed, but a new PR is okay if that's what you prefer.